app/src/pages/createcharge.tsx

```diff
@@ -44,10 +45,16 @@ const CreateChargePage: NextPage = () => {
         const billName = form.elements["name"].value;
         const billAmount = form.elements["amount"].value;
         const dueDate = form.elements["dueDate"].value;
+        const category = form.elements["category"].value;
+        if(billName.length < 1 || billAmount < 0.01 || category < 1) {
+            setError("The form elements cannot be empty or negative!");
```

Comment:
Tedious, but it would be nice if each check said what was wrong individually instead of form elements can't be empty. User might not understand what this means.